lib/cc/analyzer/formatters/plain_text_formatter.rb

```diff
@@ -7,6 +7,17 @@ module Analyzer
     module Formatters
       class PlainTextFormatter < Formatter
 
+        SEVERITY_LEVELS = [
+          INFO = "info",
```

Comment:
Does our spec allow arbitrary values for severity or do we need to validate these values?